### ISSUE TITLE
sortDirection code snippet is not coherent

### ISSUE THREAD

**Issue by mikkelrd** [DevMountain/angular-friends#14]
sortDirection needs to be a boolean. So, the `value` attribute for one of the options should be an empty string, while the other is anything other than an empty string (plain html values are always interpreted as strings, which is why ng-options is useful).

or

the orderBy syntax should be rewritten as: `sortDirection + sortProp`

**Comment by asBrettisay** [comment]
I agree, needs to be fixed. I'm wondering if empty string is the best idea, it is a shortcut but seems to be just as complex.

Is there another option that would be really clear to students what it is trying to accomplish?

**Comment by asBrettisay** [comment]
Maybe easiest would be as you suggest, change the syntax for orderBy
